webview/src/experiments/components/App.test.tsx

```diff
@@ -791,7 +791,12 @@ describe('App', () => {
       jest.advanceTimersByTime(100)
       const menuitems = screen.getAllByRole('menuitem')
       const itemLabels = menuitems.map(item => item.textContent)
-      expect(itemLabels).toStrictEqual(['Modify and Run', 'Modify and Queue'])
+      expect(itemLabels).toStrictEqual([
+        'Modify and Run',
+        'Modify and Queue',
+        'Plot',
+        'Unplot'
```

Comment:
[I] We shouldn't show both of these at the same time. Especially when one of them will not have any effect. If the experiment is selected for plotting we should show "unplot" and visa versa.